docs/2_QUICK_START_GUIDE.md

```diff
@@ -1,13 +1,229 @@
 # 2. Quick Start Guide
 
 ## 2.1 Overview
-< Describe the general context around this walkthrough/lab >
+
+*Cobalt* is a tool for developers who are interested in reusing or contributing new cloud infrastructure as code patterns in template form. An actualized infrastructure as code pattern in Cobalt is called a *Cobalt Infrastructure Template* or *CIT* (/kɪt/). Cobalt Infrastructure Templates primarily rely on [*Terraform*](https://learn.hashicorp.com/terraform)'s HCL language in order to target a wide array of cloud providers.
```

Comment:
Do you also consider our Azure DevOPS templates / patterns as infrastructure templates or more PaaS pattern kit's?

Reply:
Per our group conversation, we are currently referring to DevOPS scripts as infrastructure as code templates.